cloud/ec2_client.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 const defaultRegion = "us-east-1"
+const MockIPV6 = "1234::aba:asdf:a211:736"
```

Comment:
Is this valid?

Reply:
The format is valid, if that's what you mean.